From 55a707283dbb3b2fdcaaa4e177d050e122cfc2a0 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Mon, 29 Dec 2014 13:55:16 +0100
Subject: [PATCH 4/5] Add stack direction for the Blackfin architecture

The stack grows downward on Blackfin, see
https://sourceware.org/binutils/docs/as/Blackfin-Syntax.html: "The
stack grows by decrementing the stack pointer.".

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 m4/stack-direction.m4 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index 53f4216..1a97d5a 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -18,6 +18,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       arc | \
       arm* | strongarm* | xscale* | \
       avr | \
+      bfin | \
       c1 | c2 | c32 | c34 | c38 | \
       clipper | \
       cris | \
-- 
2.1.0

